src/extensions/default/bramble/lib/RemoteCommandHandler.js

```diff
@@ -120,6 +124,30 @@ define(function (require, exports, module) {
             skipCallback = true;
             CommandManager.execute("bramble.addFileWithType", args[0]).always(callback);
             break;
+        case "BRAMBLE_ADD_NEW_FILE_WITH_CONTENTS":
+            skipCallback = true;
+            // Add the new file to the project root, refresh the file tree, and open.
+            console.log('args', args);
+            args[0] = Path.join(StartupState.project("root"), args[0]);
+            BracketsFiler.fs().writeFile(
+                args[0],
+                new FilerBuffer(args[1]),
+                {encoding: null},
+                function(err) {
+                    if(err) {
+                        return callback(err);
+                    }
+
+                    CommandManager.execute(Commands.FILE_REFRESH)
+                        .always(function() {
+                            CommandManager.execute(
+                                Commands.CMD_ADD_TO_WORKINGSET_AND_OPEN,
```

Comment:
No, because we hide the working set.  If you just want to write a file into the fs and not trigger it to open, you'd use `fs.writeFile()` in the hosting app.  Do you want to make it so you can configure whether or not to open the file after it's written?  I could do that via an arg.